tools/mason/MasonHelp.chpl

```diff
@@ -353,6 +353,8 @@ proc masonTestHelp() {
   writeln("    -h, --help                  Display this message");
   writeln("        --show                  Direct output of tests to stdout");
   writeln("        --no-run                Compile tests without running them");
+  writeln("        --keep-binary           Doesn't delete the binaries after running");
+  writeln("        --subdir                Descend recursively into subdirectories of given Directories");
   writeln("        --parallel              Run tests in parallel(sequential by default)");
   writeln();
```

Comment:
We need to specify that `mason test` works inside and outside of mason packages.

Reply:
I have made some changes. Would be helpful if you can look at them.